app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -2,40 +2,85 @@ module Katello
   module Concerns
     module BaseTemplateScopeExtensions
       extend ActiveSupport::Concern
+      extend ApipieDSL::Module
 
+      apipie :class, 'Base macros related to content hosts to use within a template' do
+        name 'Base Content'
+        sections only: %w[all reports provisioning jobs partition_tables]
+      end
+
+      # TODO: need help
+      # apipie :method, 'Returns ?' do
+      #   desc ''
+      #   required :id, Integer, desc: 'errata id?'
+      #   returns Object
+      # end
       def errata(id)
         Katello::Erratum.in_repositories(Katello::Repository.readable).with_identifiers(id).map(&:attributes).first.slice!('created_at', 'updated_at')
       end
 
+      apipie :method, 'Returns subscriptions for the host' do
+        required :host, 'Host::Managed', desc: 'Host object to get subscriptions for'
+        # returns array_of: 'Subscription' TODO: needs Subscription class to be documented
```

Comment:
Not sure which class needs to be documented though. The first thought is app/models/katello/subscription.rb. But the class doesn't have `Jail` with supported methods in safe-mode rendering. I could add basic `id/name` if this is okay.

Reply:
+1 i think a basic jail with id/name is fine.  we can always add more later